cassandra-analytics-integration-tests/src/test/java/org/apache/cassandra/sidecar/testing/IntegrationTestBase.java

```diff
@@ -0,0 +1,234 @@
+        assertThat(session).isNotNull();
+        return session;
+    }
+
+    private static QualifiedTableName uniqueTestTableFullName()
+    {
+        return new QualifiedTableName(TEST_KEYSPACE, TEST_TABLE_PREFIX + TEST_TABLE_ID.getAndIncrement());
+    }
+
+    public List<Path> findChildFile(CassandraSidecarTestContext context, String hostname, String target)
+    {
+        InstanceMetadata instanceConfig = context.instancesConfig().instanceFromHost(hostname);
+        List<String> parentDirectories = instanceConfig.dataDirs();
+
+        return parentDirectories.stream().flatMap(s -> findChildFile(Paths.get(s), target).stream())
+                                .collect(Collectors.toList());
+    }
+
+    private List<Path> findChildFile(Path path, String target)
+    {
+        try (Stream<Path> walkStream = Files.walk(path))
+        {
+            return walkStream.filter(p -> p.toString().endsWith(target)
+                                          || p.toString().contains("/" + target + "/"))
+                             .collect(Collectors.toList());
+        }
+        catch (IOException e)
+        {
+            return Collections.emptyList();
+        }
+    }
+
+    private static void healthCheck(InstancesConfig instancesConfig)
+    {
+        instancesConfig.instances()
+                       .forEach(instanceMetadata -> instanceMetadata.delegate().healthCheck());
+    }
+
+    /**
+     * Waits for the specified keyspace/table to be available.
+     * Emperically, this loop usually executes either zero or one time before completing.
+     * However, we set a fairly high number of retries to account for variability in build machines.
+     *
+     * @param keyspaceName the keyspace for which to wait
+     * @param tableName    the table in the keyspace for which to wait
+     */
+    protected void waitForKeyspaceAndTable(String keyspaceName, String tableName)
+    {
+        int numInstances = sidecarTestContext.instancesConfig().instances().size();
+        int retries = MAX_KEYSPACE_TABLE_WAIT_ATTEMPTS;
+        boolean sidecarMissingSchema = true;
+        while (sidecarMissingSchema && retries-- > 0)
+        {
+            sidecarMissingSchema = false;
+            for (int i = 0; i < numInstances; i++)
+            {
+                KeyspaceMetadata keyspace = sidecarTestContext.session(i)
+                                                              .getCluster()
+                                                              .getMetadata()
+                                                              .getKeyspace(keyspaceName);
+                sidecarMissingSchema |= (keyspace == null || keyspace.getTable(tableName) == null);
+            }
+            if (sidecarMissingSchema)
+            {
+                logger.info("Keyspace/table {}/{} not yet available - waiting...", keyspaceName, tableName);
+                Uninterruptibles.sleepUninterruptibly(100, TimeUnit.MILLISECONDS);
```

Comment:
Nit: define a constant for the wait time too, and place it next to `MAX_KEYSPACE_TABLE_WAIT_ATTEMPTS`, so that the comment is more straightforward.